Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.cpp

```diff
@@ -910,7 +910,7 @@ void connectSimpleBusMessageCallback(GstElement* pipeline, Function<void(GstMess
 template<>
 Vector<uint8_t> GstMappedBuffer::createVector() const
 {
-    return std::span<const uint8_t> { data(), size() };
+    return unsafeMakeSpan<const uint8_t>(data(), size());
```

Comment:
There's a `GstBufferMapper::span()` function, which is inherited by `GstMappedBuffer`:

```suggestion
    return Vector(span());
```